opam

```diff
@@ -21,7 +21,7 @@ remove: [
 depends: [
   "cstruct" {>= "1.0.1"}
   "ounit" {test}
-  "ocamlbuild" {build}
+  "oasis" {build}
```

Comment:
you actually need both `ocamlbuild` and also `ocamlfind` during `{build}` (as well as `oasis`)...